tests/core/pyspec/eth2spec/test/helpers/withdrawals.py

```diff
@@ -295,26 +301,63 @@ def run_withdrawals_processing(
     yield "post", state
 
     # Check withdrawal indices
-    assert state.next_withdrawal_index == pre_state.next_withdrawal_index + len(
```

Comment:
@terencechain I got the impression that because there is a func called `verify_post_state` we should move all the post_state assertions there. I will add that